common/src/test/java/com/revenuecat/purchases/common/AppConfigTest.kt

```diff
@@ -166,14 +177,16 @@ class AppConfigTest {
             observerMode = false,
             platformInfo = PlatformInfo(flavor = "native", version = "3.2.0"),
             proxyURL = null,
-            store = Store.PLAY_STORE
+            store = Store.PLAY_STORE,
+            dangerousSettings = DangerousSettings(autoSyncPurchases = true)
```

Comment:
What do you think about adding `DangerousSettings.DEFAULT` to avoid the repetition?

Reply:
Oh, I can just do `DangerousSettings()`, since `autoSyncPurchases = true` is the default. Does that work? I can also add a default to the constructor of `AppConfig`